app/assets/javascripts/models/beta/story.js

```diff
@@ -118,14 +118,19 @@ export const editStory = (story, newAttributes) => {
   };
 };
 
-export const deserialize = (story) => ({
-  ...story,
-  labels: Label.splitLabels(story.labels),
-  estimate: story.estimate || '',
-  collapsed: true
-});
+export const deserialize = (story, collapse) => {
+  const collapsed = collapse === false ? false : true;
```

Comment:
Why that verification? It seems redundant.

Reply:
It's a optional parameter, so sometimes it can be `undefined`, so just `collapse = collapse || true;` don't solve my problem. :disappointed:

Reply:
Can you use a default parameter value to solve this? 🤔 
```js
export const deserialize = (story, collapse = true) => {
```

Reply:
I have tried to do like this, but it isn't working. I think our babel is out of date :thinking:. Need to update this in a another task.

Reply:
Really fancier @luciamo :smile: but in some cases, it will be `undefined` and in this case, I need to `collapse = true` and `!!undefined` is equal to `false`. :(

Reply:
So, the values you're receiving are `true`, `false` and `undefined`, right?
I believe you can use this code below:

```suggestion
  const collapsed = collapse === undefined ? true : collapse;
```

Reply:
Actually is just `false` or `undefined`, but yes, this way is better.